src/task2.java

```diff
@@ -0,0 +1,98 @@
+public class task2 {
+    public task2() {
+    }
+
+    static void stringToArrayToArrayToString() {
```

Comment:
this method can be simplified by using a single loop to reverse the array, instead of using an additional array and complex logic